Drop-In/src/main/java/com/braintreepayments/api/DropInEvent.java

```diff
@@ -0,0 +1,107 @@
+package com.braintreepayments.api;
+
+import android.os.Bundle;
+import android.os.Parcel;
+import android.os.Parcelable;
+
+class DropInEvent {
+
+    static final String REQUEST_KEY = "DROP_IN_EVENT_REQUEST_KEY";
+
+    private static final String TYPE_KEY = "DROP_IN_EVENT_TYPE";
+
+    static DropInEvent createSendAnalyticsEvent(String eventName) {
+        DropInEvent event = new DropInEvent(DropInEventType.SEND_ANALYTICS);
+        event.putString(DropInEventProperty.ANALYTICS_EVENT_NAME, eventName);
+        return event;
+    }
+
+    static DropInEvent createAddCardSubmitEvent(String cardNumber) {
+        DropInEvent event = new DropInEvent(DropInEventType.ADD_CARD_SUBMIT);
+        event.putString(DropInEventProperty.CARD_NUMBER, cardNumber);
+        return event;
+    }
+
+    static DropInEvent createCardDetailsSubmitEvent(Card card) {
+        DropInEvent event = new DropInEvent(DropInEventType.CARD_DETAILS_SUBMIT);
+        event.putParcelable(DropInEventProperty.CARD, card);
+        return event;
+    }
+
+    static DropInEvent createEditCardEvent(String cardNumber) {
```

Comment:
Take it or leave it comment 
```suggestion
    static DropInEvent createEditCardNumberEvent(String cardNumber) {
```

Reply:
Def!